librz/util/buf.c

```diff
@@ -408,7 +515,16 @@ RZ_API RzBuffer *rz_buf_new_with_methods(RZ_NONNULL const RzBufferMethods *metho
 	return b;
 }
 
-RZ_API RzBuffer *rz_buf_new_with_pointers(const ut8 *bytes, ut64 len, bool steal) {
+/**
+ * \brief Creates a new buffer with a bytes array.
+ * \param bytes The bytes array used to initialized the buffer.
+ * \param len The length of the bytes array.
+ * \param steal If the boolean is true the function take the ownership of the data.
+ * \return Return the new allocated buffer.
+ *
+ * The function creates a new allocated buffer using the RZ_BUFFER_BYTES back end.
```

Comment:
The function creates a new buffer in memory. The argument \p bytes is used as the memory backend if \p steal is true, otherwise a new buffer is allocated and its content is initialized with the one from \p bytes.